Vagrantfile

```diff
@@ -26,7 +26,7 @@ Vagrant.configure('2') do |config|
       type: 'nfs',
       create: true
     }
-    dev_config.vm.synced_folder '../www', '/home/vagrant/www', options
+    dev_config.vm.synced_folder vconfig['sync_folder_path'], '/home/vagrant/www', options
```

Comment:
I don't know if is the right way, I think we could do this in other way with existing vagrant variables, but I thinks this is a useful feature for the VM.